spk/pyload/src/app/config

```diff
@@ -0,0 +1,12 @@
+{
+    ".url": {
+        "com.synocommunity.packages.pyload": {
+            "title": "pyLoad",
+            "desc": "pyLoad",
+            "icon": "images/pyload-{0}.png",
+            "type": "url",
+            "url": "3rdparty/pyload/pyload.cgi",
+            "allUsers": true
+        }
```

Comment:
Maybe you'd want to add a `"grantPrivilege": local` to keep the same behaviour as we use for other packages.